app/src/main/java/com/example/garden_planner/adapters/ReminderAdapter.java

```diff
@@ -91,33 +103,32 @@ public void onClick(View v) {
                 Garden garden = reminder.getRemindWhat();
                 MainActivity activity = (MainActivity) context;
                 activity.goToDetailGardenView(garden);
-                // TODO: go to the detail view of garden
             }
         }
 
-        public void bind(Reminder reminder){
-            Log.i(TAG, "binding "+reminder.getReminderTitle());
+        public void bind(Reminder reminder) {
+            Log.i(TAG, "binding " + reminder.getReminderTitle());
             tvGardenName = binding.tvGardenName;
             tvToDoDate = binding.tvToDoDate;
             ivPlantPic = binding.ivPlantPic;
             tvPlantName = binding.tvPlantName;
             tvReminderTitle = binding.tvReminderTitle;
             tvReminderText = binding.tvReminderText;
+            completeReminderButton = binding.completeReminderButton;
 
-            Log.i(TAG, "binding "+reminder.getReminderTitle());
+            Log.i(TAG, "binding " + reminder.getReminderTitle());
 
             tvGardenName.setText(reminder.getRemindWhat().getName());
 
-            if (reminder.getReminderStart().equals(reminder.getReminderEnd())){
+            if (reminder.getReminderStart().equals(reminder.getReminderEnd())) {
                 tvToDoDate.setText(reminder.getReminderStart().toString());
-            }
-            else {
+            } else {
                 tvToDoDate.setText(reminder.getReminderStart().toString() + " to " + reminder.getReminderEnd().toString());
             }
 
-            if(reminder.has(Reminder.KEY_REMIND_WHICH_PLANT)) {
-                if(reminder.getRemindWhichPlant().has(PlantInBed.KEY_TYPE)){
-                    if(reminder.getRemindWhichPlant().getPlantType().has(Plant.KEY_PHOTO)){
+            if (reminder.has(Reminder.KEY_REMIND_WHICH_PLANT)) {
+                if (reminder.getRemindWhichPlant().has(PlantInBed.KEY_TYPE)) {
+                    if (reminder.getRemindWhichPlant().getPlantType().has(Plant.KEY_PHOTO)) {
```

Comment:
I tried that, but it can't because if it doesn't have the which plant key, it'll throw an error when it gets to the "has plant in bed key" - it does looks ugly, so I'll add a comment explaining it there